com/truckmuncher/api/trucks.proto

```diff
@@ -77,6 +86,18 @@ message ActiveTrucksResponse {
          * Value in the range [-180, 180]
          */
         required double longitude = 3;
+        /** 
+         * Indicates whether the truck's location is verified or not.
+         */
+        optional bool verified = 4 [default = false];
```

Comment:
Yep, this is the verified badge. We haven't really thought about being verified after X number of upvotes. Right now I don't think we would do that, but that doesn't mean we couldn't in the future.